apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -147,6 +149,23 @@ class SectionProgress extends Component {
       (currentView === ViewType.SUMMARY || currentView === ViewType.DETAIL);
     const standardsStyle =
       currentView === ViewType.STANDARDS ? styles.show : styles.hide;
+
+    // Reorder coursesWithProgress so that the current section is at the top and other sections are in order from newest to oldest
+    // Also adds "(Current Section)" label to the course version in the currently viewed section.
+    if (coursesWithProgress[0].id !== sectionVersionId) {
+      let currentCourse;
+      currentCourse = coursesWithProgress.splice(
+        coursesWithProgress.findIndex(course => {
+          return course.id === sectionVersionId;
+        }),
+        1
+      )[0];
+      currentCourse.display_name =
+        '(Current Section) ' + currentCourse.display_name;
+      coursesWithProgress.push(currentCourse);
+      coursesWithProgress.reverse();
```

Comment:
Why reverse all the order of all the other courses in the list?

Reply:
They're initially ordered chronologically. This puts them in reverse chronological order, so that the most recent section is at the top.